src/lib/API/api.js

```diff
@@ -36,8 +36,9 @@ export const getErrorMessage = apiError => {
   // if the json or string http body was thrown from axios (error
   // interceptor in api.js doest that in almost cases) then we're wrapping
   // it onto Error object to keep correct stack trace for Sentry reporting
+  // stringify apiError object before passing to the Error constructor, to prevent getting error message [object Object]
   if (!isError(apiError)) {
-    message = apiError.error || apiError
+    message = apiError.error || JSON.stringify(apiError)
```

Comment:
what are other cases for apiError ? if it's goodserver response it could also be an object { success: false, error: <some string> }, let's handle this case & use apiError.error as the message.

Reply:
it always will be a server response - and we already handle 'apiError.error' then if there's no 'apiError.error' prop just stringify the hole object to have some error info in error logs

Reply:
@yaroslav-fedyshyn-nordwhale which are the cases when there's no .error prop ? zoomserver should always return { error, success: false } on failure except the cases when 404 or 50x error happened and body is a string "Request faield with code XXX"

Reply:
I am not sure about the exact cases. But there are some logs in sentry with eMsg [object Object] which appear here.
It means that there are not error prop inside apiError object.

Reply:
@yaroslav-fedyshyn-nordwhale pls reproduce those errors and check what exactly comes to apiError

Reply:
done